packages/bitgenerators/bitgenerators.v0.1.0/opam

```diff
@@ -0,0 +1,45 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "PRNG bitgenerators for OCaml users"
+description:
+  "A purely functional port of numpy's random bitgenerator interface for OCaml users"
+maintainer: ["Zolisa Bleki"]
+authors: ["Zolisa Bleki"]
+license: "BSD-3-Clause"
+tags: ["scientific computing" "psuedo-random numbers"]
+homepage: "https://github.com/zoj613/bitgenerators"
+doc: "https://zoj613.github.io/bitgenerators/bitgenerators/Bitgen/index.html"
+bug-reports: "https://github.com/zoj613/bitgenerators/issues"
+depends: [
+  "ocaml" {>= "4.14.0"}
+  "dune" {>= "3.14"}
+  "ctypes"
+  "stdint"
+  "odoc" {with-doc}
+  "bisect_ppx" {dev & >= "2.5.0" & with-test}
+  "ounit2" {with-test}
+  "csv" {with-test}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/zoj613/bitgenerators.git"
```

Comment:
```suggestion
dev-repo: "git+https://github.com/zoj613/bitgenerators.git"
conflicts: [ "ocaml-option-bytecode-only" ]
```

Reply:
This should fix the 32 bit issue

Reply:
Unfortunately, that does not seem to fix the 2 failures on arm32 and x86_32. See here: https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/5229ac482af57d7b360a851ffc1048f12c0d7966